src/main/java/org/mifos/dpgtemplate/conductorsampleworkers/AddNumbersWorker.java

```diff
@@ -0,0 +1,31 @@
+package org.mifos.dpgtemplate.conductorsampleworkers;
+
+import com.netflix.conductor.client.worker.Worker;
+import com.netflix.conductor.common.metadata.tasks.Task;
+import com.netflix.conductor.common.metadata.tasks.TaskResult;
+
+public class AddNumbersWorker implements Worker {
+
+    private final String taskDefName;
+
+    public AddNumbersWorker(String taskDefName) {
+        this.taskDefName = taskDefName;
+    }
+
+    @Override
+    public String getTaskDefName() {
+        return taskDefName;
+    }
+
+    @Override
+    public TaskResult execute(Task task) {
+        TaskResult result = new TaskResult(task);
```

Comment:
Not very sure whether the sample code can created conflicts when added to multiple services and missed to be removed.